Formula/t/tigerbeetle.rb

```diff
@@ -0,0 +1,63 @@
+class Tigerbeetle < Formula
+  desc "Financial transactions database"
+  homepage "https://tigerbeetle.com"
+  url "https://github.com/tigerbeetle/tigerbeetle.git",
+      tag:      "0.16.27",
+      revision: "1c7db9310d65ef9b8dcef61431cd4d8b6da95459"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+  depends_on "zig" => :build
+
+  on_linux do
+    depends_on "liburing"
+  end
+
+  def install
+    # Fix illegal instruction errors when using bottles on older CPUs.
+    # https://github.com/Homebrew/homebrew-core/issues/92282
+    case Hardware.oldest_cpu
+    when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+    else Hardware.oldest_cpu
+    end
+
+    args = %W[
+      --prefix #{prefix}
+      -Drelease
+      -Dconfig-release=#{version}
+      -Dconfig-release-client-min=0.15.3
```

Comment:
Where does this come from?

Reply:
Just the release notes at the moment. I'm not sure we can do anything to collect it systematically - it seems to be used to keep track of breaking changes - is required to be passed when `config-release` is set.